common/config/presets/tari_config_example.toml

```diff
@@ -119,6 +119,8 @@ base_node_query_timeout = 120
 # (options: "DirectOnly", "StoreAndForwardOnly", DirectAndStoreAndForward". default: "DirectAndStoreAndForward").
 #transaction_routing_mechanism = "DirectAndStoreAndForward"
 
+scan_for_utxo_interval=60
```

Comment:
In my opinion, this is much too frequent. I think a really aggressive setting would be 10 minutes.

Reply:
My view is to use this short duration to help debug performance and other problems, and we can increase it later